drape_frontend/rule_drawer.cpp

```diff
@@ -363,6 +363,7 @@ void RuleDrawer::ProcessLineStyle(FeatureType & f, Stylist const & s,
 
   if (needAdditional && !clippedSplines.empty())
   {
+    minVisibleScale = feature::GetMinDrawableScale(f);
```

Comment:
Why? Is it differ with ApplyLineFeatureGeometry calculation?

Reply:
There is no minVisibleScale calculation in `ApplyLineFeatureGeometry`.
But maybe its actually needed there also... Though in #4428 path_text received their minVisibleScale values from `ApplyLineFeatureAdditional`.. I'll take a closer look.

Reply:
Ah, yes. I was confused that RuleDrawer::ProcessLineStyle takes minVisibleScale by reference.
So, need to investigate the difference between:
- minVisibleScale calculation in RuleDrawer::operator()
- feature::GetMinDrawableScale()

And remove passing it by reference ..

Reply:
I double-checked:
`ApplyLineFeatureGeometry` process line primitives and line symbols (i.e. oneway arrows) only, and both of them don't use minVisibleScale at all.

`RuleDrawer::operator()` inits minVisibleScale to 0 and then passes it by reference to e.g. point/area/line processors, where it could be set to a real value taken from `feature::GetMinDrawableScale()` if necessary.

So my code is correct here - it adds a calculation of real minVisibleScale value for line processing only when its really needed (e.g. road shields, path_texts...)

Reply:
I will check this